draft-ietf-ppm-dap.md

```diff
@@ -1524,82 +1556,108 @@ the requirements in {{request-authentication}}.
 ## Collecting Results {#collect-flow}
 
 In this phase, the Collector requests aggregate shares from each aggregator and
-then locally combines them to yield a single aggregate result. In particular,
-the Collector issues a query to the Leader ({{query}}), which the Aggregators
-use to select a batch of reports to aggregate. Each emits an aggregate share
-encrypted to the Collector so that it can decrypt and combine them to yield the
-aggregate result. This entire process is composed of two interactions:
-
-1. Collect request and response between the collector and leader, specified in
-   {{collect-init}}
-1. Aggregate share request and response between the leader and each aggregator,
-   specified in {{collect-aggregate}}
+then locally combines them to yield a single aggregate result. This interaction
+is mediated through the Leader, which is responsible for obtaining aggregate
+shares from the Helper and relaying them to the Collector. In particular, the
+Collector issues a query to the Leader ({{query}}), which the Aggregators use to
+select a batch of reports to aggregate. Each emits an aggregate share encrypted
+to the Collector so that it can decrypt and combine them to yield the aggregate
+result. This entire process revolves around two resources:
+
+1. The Collector uses the collections resource exposed by the Leader (see
+   {{collection-resource}}).
+1. The Leader uses the aggregate shares resource exposed by the Helper (see
+   {{aggregate-share-resource}}).
 
 Once complete, the collector computes the final aggregate result as specified in
 {{collect-finalization}}.
 
-### Collection Initialization {#collect-init}
+### Collection Resource {#collection-resource}
 
-To initiate collection, the collector issues a POST request to
-`[leader]/collect`, where `[leader]` is the leader's endpoint URL. The body of
-the request is structured as follows:
+This resource allows the Collector to create new collections in the Leader and
```

Comment:
I don't recall if we agreed to be consistent about this, but just in case:
```suggestion
This resource allows the Collector to create new collection jobs in the Leader and
```

Reply:
I prefer "collections" over "collection jobs". The collection _job_ is the means by which we get the thing we actually care about, which is the collection. But at the end of the day, it doesn't really make a difference which string we use here.